core/modules/link/link.module

```diff
@@ -0,0 +1,1460 @@
+<?php
+
+/**
+ * @file
+ * Defines simple link field types.
+ */
+
+define('LINK_EXTERNAL', 'external');
+define('LINK_INTERNAL', 'internal');
+define('LINK_FRONT', 'front');
+define('LINK_EMAIL', 'email');
+define('LINK_NEWS', 'news');
+define('LINK_DOMAINS', 'aero|arpa|asia|biz|build|com|cat|ceo|coop|edu|gov|info|int|jobs|mil|museum|name|nato|net|org|post|pro|tel|travel|mobi|local|xxx');
+
+define('LINK_TARGET_DEFAULT', 'default');
+define('LINK_TARGET_NEW_WINDOW', '_blank');
+define('LINK_TARGET_TOP', '_top');
+define('LINK_TARGET_USER', 'user');
+
+/**
+* Implements hook_config_info().
+*/
+function link_config_info() {
+  $prefixes['link.settings'] = array(
+    'label' => t('Settings'),
+    'group' => t('Link'),
+  );
+  return $prefixes;
+}
+
+/**
+ * Maximum URLs length - needs to match value in link.install.
+ */
+define('LINK_URL_MAX_LENGTH', 2048);
+
+/**
+ * Implements hook_field_info().
+ */
+function link_field_info() {
+  return array(
+    'link_field' => array(
+      'label' => t('Link'),
+      'description' => t('Store a title, href, and attributes in the database to assemble a link.'),
+      'settings' => array(
+        'attributes' => _link_default_attributes(),
+        'url' => 0,
+        'title' => 'optional',
+        'title_value' => '',
+        'title_maxlength' => 128,
+        'enable_tokens' => 1,
+        'display' => array(
+          'url_cutoff' => 80,
+        ),
+      ),
```

Comment:
ok; commented out the settings array elements and flagged with `PORT_TO_BACKDROP_CORE` todo comment.

```php
'settings' => array(
        // PORT_TO_BACKDROP_CORE
        //TODO:  working hypothesis is that
        // all link field are instance settings and we don't need these.
        /*'attributes' => _link_default_attributes(),
        'url' => 0,
        'title' => 'optional',
        'title_value' => '',
        'title_maxlength' => 128,
        'enable_tokens' => 1,
        'display' => array(
          'url_cutoff' => 80,
        ),*/
      ),
```